tests/pytest_tests/unit_tests/test_dtypes.py

```diff
@@ -758,10 +758,12 @@ def test_table_typing_numpy():
     table.add_data(np.uintp(42))
     table.add_data(np.float32(42))
     table.add_data(np.float64(42))
-    table.add_data(np.float_(42))
     table.add_data(np.complex64(42))
     table.add_data(np.complex128(42))
-    table.add_data(np.complex_(42))
+
+    if np.__version__.startswith("1."):
+        table.add_data(np.float_(42))
+        table.add_data(np.complex_(42))
```

Comment:
we only use a single version in CI, so this is either dead code or an unnecessary if-statement, and it's not clear which